src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs

```diff
@@ -134,1041 +130,31 @@ internal partial class StringsHelper
-            internal const string ADP_InvalidMixedUsageOfAccessTokenAndIntegratedSecurity = @"Cannot set the AccessToken property if the 'Integrated Security' connection string keyword has been set to 'true' or 'SSPI'.";
-            internal const string ADP_InvalidMixedUsageOfAccessTokenAndContextConnection = @"Cannot set the AccessToken property with the 'Context Connection' keyword.";
-            internal const string ADP_InvalidMixedUsageOfAccessTokenAndCredential = @"Cannot set the AccessToken property if the Credential property is already set.";
-            internal const string ADP_InvalidMixedUsageOfCredentialAndAccessToken = @"Cannot set the Credential property if the AccessToken property is already set.";
-            internal const string ADP_InvalidMixedUsageOfAccessTokenAndAuthentication = @"Cannot set the AccessToken property if 'Authentication' has been specified in the connection string.";
-            internal const string ADP_MustBeReadOnly = @"{0} must be marked as read only.";
             internal const string DataCategory_Data = @"Data";
-            internal const string DataCategory_StateChange = @"StateChange";
             internal const string DataCategory_Update = @"Update";
             internal const string DbCommand_CommandTimeout = @"Time to wait for command to execute.";
             internal const string DbConnection_State = @"The ConnectionState indicating whether the connection is open or closed.";
-            internal const string DbConnection_StateChange = @"Event triggered when the connection changes state.";
-            internal const string DbParameter_DbType = @"The parameter generic type.";
-            internal const string DbParameter_Direction = @"Input, output, or bidirectional parameter.";
-            internal const string DbParameter_IsNullable = @"a design-time property used for strongly typed code-generation.";
-            internal const string DbParameter_Offset = @"Offset in variable length data types.";
-            internal const string DbParameter_ParameterName = @"Name of the parameter.";
-            internal const string DbParameter_Size = @"Size of variable length data types (string & arrays).";
-            internal const string DbParameter_SourceColumn = @"When used by a DataAdapter.Update, the source column name that is used to find the DataSetColumn name in the ColumnMappings. This is to copy a value between the parameter and a data row.";
-            internal const string DbParameter_SourceVersion = @"When used by a DataAdapter.Update (UpdateCommand only), the version of the DataRow value that is used to update the data source.";
-            internal const string DbParameter_SourceColumnNullMapping = @"When used by DataAdapter.Update, the parameter value is changed from DBNull.Value into (Int32)1 or (Int32)0 if non-null.";
-            internal const string DbParameter_Value = @"Value of the parameter.";
```

Comment:
Could you include `DbParameter` const strings?

Reply:
Did I miss - they are used anywhere?

Reply:
My guess is they're for `SqlParameter`.

Reply:
I only found one "DbParameter_SourceColumnNullMapping" used in [SqlParameter](https://github.com/dotnet/SqlClient/blob/d1ff708949999bd8c02991beec19f3ac6c9540fe/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlParameter.cs#L840) which I'll include, but it's not used in NetFx and no other properties have description attributes, so there's inconsistency in there.